docs/guides/chip_tool_guide.md

```diff
@@ -1393,3 +1393,244 @@ $ ./chip-tool any read-by-id <cluster-ids> <attribute-ids> <destination-id> <end
+```
+
+### Step 2: Assign a credential
+
+Once you have a user created on the door lock device, use the following command
+pattern to assign a credential to it:
+
+```
+$ ./chip-tool doorlock set-credential <operation-type> <{Credential}> <credential-data> <user-index> <user-status> <user-type> <destination-id> <endpoint-id> --timedInteractionTimeoutMs <ms_value>
+```
+
+In this command:
+
+-   _<operation-type\>_ is one of the available types of operation for the
+    credential:
+
+    -   `Add` - This operation adds a new credential to a user at the slot at
+        _<user-index\>_.
+    -   `Clear` - This operation removes an existing credential from the user at
+        the slot at _<user-index\>_.
+    -   `Modify` - This operation modifies an existing credential for the user
+        at the slot at _<user-index\>_.
+
+-   _<{Credential}\>_ is a parameter in the JSON format, with the following two
+    fields:
+
+    -   _<credential-type\>_ is the type of the credential. It can have one of
+        the following values:
+
+        -   `0` - Programming PIN
+        -   `1` - PIN
+        -   `2` - RFID
+        -   `3` - Fingerprint
+        -   `4` - Finger vein
+
+    -   _<credential-index\>_ is the index of the credential, between `1` and
+        the maximum value you set in the `NumberOfCredentialsSupportedPerUser`
+        attribute (see the section 5.2.3.20 of the Matter Application Clusters
+        specification for details). Setting the credential index to `0` will
+        cause an error.
+
+-   _<credential-data\>_ is a string parameter with the secret credential data.
+    For example, the PIN code value (`12345` in the example below).
+-   _<user-index\>_ is the index of the user that will be associated with the
+    credential. Can be set to `null`.
+-   _<user-status\>_ is the status of the user that will be associated with the
+    credential. See the description of this parameter in
+    [Set up a user](#step-1-set-up-a-user). Can be set to `null`.
+-   _<destination-id\>_ is the ID of the door lock device.
+-   _<endpoint-id\>_ is the ID of the endpoint on the door lock device.
+-   `--timedInteractionTimeoutMs` is the timeout duration for the interaction,
+    in milliseconds (_<ms_value>_). This should allow enough time for the
+    interaction to complete.
+
+**Example of command:**
+
+```
+$ ./chip-tool doorlock set-credential 0 '{ "credentialType": 1, "credentialIndex": 1 }' "12345" 1 null null 1 1 --timedInteractionTimeoutMs 1000
+```
+
+### Operations on users and credentials
+
+After you set up users and credentials on your door lock device, you can use
+several CHIP Tool commands to interact with them.
+
+All commands reuse the parameters explained earlier in this section. The
+following command patterns are available:
+
+-   Reading the status of the user:
+
+    ```
+    $ ./chip-tool doorlock get-user <user-index> <destination-id> <endpoint-id> --timedInteractionTimeoutMs <ms_value>
```

Comment:
Note that get-user does not need `--timedInteractionTimeoutMs` at all.  Take it out here, please.
```suggestion
    $ ./chip-tool doorlock get-user <user-index> <destination-id> <endpoint-id>
```